src/library/baseplaylistfeature.cpp

```diff
@@ -478,6 +483,102 @@ void BasePlaylistFeature::slotExportPlaylist() {
     }
 }
 
+void BasePlaylistFeature::slotExportTrackFiles() {
+    // TODO: This is a near-exact duplication of the same code in CreateFeature.
+    // Factor it out!
```

Comment:
Or since this is specific to the library, just put it in library/soundfileexport.cpp